File: html/redcap_v15.7.6/API/examples/python/export_survey_return_code.py
#!/usr/bin/env python

from config import config
import requests

fields = {
    'token': config['api_token'],
    'content': 'surveyReturnCode',
    'record': 'f21a3ffd37fc0b3c',
    'instrument': 'test_instrument',
    'event': 'event_1_arm_1',
    'format': 'json'
}

r = requests.post(config['api_url'],data=fields)
print('HTTP Status: ' + str(r.status_code))
print(r.text)

File: html/redcap_v15.7.6/API/examples/perl/export_survey_return_code.pl
#!/usr/bin/env perl

use strict;
use warnings;
use LWP::Curl;

my %config = do 'config.pl';

my $fields = {
    token      => $config{api_token},
    content    => 'surveyReturnCode',
    record     => 'f21a3ffd37fc0b3c',
    instrument => 'test_instrument',
    event      => 'event_1_arm_1',
    format     => 'json'
};

my $ch = LWP::Curl->new(auto_encode => 0);
my $content = $ch->post($config{api_url}, $fields, $config{referer});

print $content;
